blogposts/2021/release-3.32.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Sourcegraph 3.32 release'
+publishDate: 2021-09-23T10:00-07:00
+description: 'Sourcegraph 3.32 introduces Code Insights in Public Beta, a new search sidebar for navigating revisions, improved search results caching, and custom LSIF data retention policies.'
+tags: [blog, release]
+slug: 'release/3.32'
+published: false
+heroImage:
+socialImage:
+changelogItems:
+  - description: The search sidebar now shows a revisions section when all search results are from a single repository. This makes it easier to search in and switch between different revisions.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23835
+    category: Search
+  - description: The various alerts overview panels in Grafana can now be clicked to go directly to the relevant panels and dashboards.
+    url: https://github.com/sourcegraph/sourcegraph/pull/24920
+    category: Admin
+  - description: '`allowGroupsPermissionsSync` in the GitHub authorization provider is now required to enable the experimental GitHub teams and organization permissions caching.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/24561
+    category: Admin
+  - description: Fixed a number of issues where repository permissions sync may fail for instances with very large numbers of repositories.
+    url: https://github.com/sourcegraph/sourcegraph/pull/24972
+    category: Repositories
+  - description: 'Fixed excessive re-rendering of the web application on every keypress in the search query input.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/24844
+    category: Search
+  - description: 'The `PRECISE_CODE_INTEL_DATA_TTL` environment variable is no longer read by the worker service. Instead, global and repository-specific data retention policies configurable in the UI by site-admins will control the length of time LSIF uploads are considered _fresh_.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/24793
+    category: Admin
+---
+
+Sourcegraph 3.32 is now available! Here are some highlights from this release:
+
+## Code Insights is available in Public Beta
+
+Code Insights is a new analytics tool that lets you track and understand what’s in your code and how it changes over time. You can find it via “Insights” in the main nav bar. It is free while in beta through 2021. When Code Insights is officially released, we may disable your use of it or charge for using it. We're still polishing Code Insights, so we appreciate if you [share any bugs or feedback about your experience](mailto:feedback@sourcegraph.com).
```

Comment:
```suggestion
Code Insights is a new analytics tool that lets you track and understand what’s in your code and how it changes over time. You can find it via “Insights” in the main navigation bar. It is available for all Sourcegraph users for free while in beta. When Code Insights reaches general availability, we may disable your use of it or charge for using it. We're still polishing Code Insights, so we appreciate it if you [share any bugs or feedback about your experience](mailto:feedback@sourcegraph.com).
```

Reply:
@thisisvic @Joelkw — I proposed a couple of edits here. I would keep some info higher level and remove "through 2021". How do you feel about this?

Reply:
(Updated suggestion)

Reply:
Thank you, @iskyOS! It looks really good to me and flows nicely. @Joelkw how do you feel about the tweaks?

Reply:
Fine with it!